tests/core/full_node/full_sync/test_full_sync.py

```diff
@@ -6,7 +6,9 @@
 
 import pytest
 
+from chia.full_node.weight_proof import _validate_sub_epoch_summaries
```

Comment:
As per convention the somewhere imported function should probably not have the _ prefix anymore?

Reply:
i think its ok since its a test file